novaan-server/src/Auth/Jwt/JwtService.cs

```diff
@@ -0,0 +1,166 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
```

Comment:
The type or namespace name 'IdentityModel' does not exist in the namespace 'Microsoft'